src/contracts/interfaces/IBLSRegistryCoordinatorWithIndices.sol

```diff
@@ -27,12 +27,13 @@ interface IBLSRegistryCoordinatorWithIndices is ISignatureUtils, IRegistryCoordi
     }
 
     /**
-     * @notice Data structure for the parameters needed to kick an operator from a quorum, used during registration churn.
+     * @notice Data structure for the parameters needed to kick an operator from a quorum with number `quorumNumber`, used during registration churn.
      * Specifically the `operator` is the address of the operator to kick, `pubkey` is the BLS public key of the operator,
      * `operatorIdsToSwap` is the list of operatorIds to swap with the operator being kicked in the indexRegistry,
      * and `globalOperatorListIndex` is the index of the operator in the global operator list in the indexRegistry.
      */
     struct OperatorKickParam {
+        uint8 quorumNumber;
```

Comment:
why is this a singlet but `operatorIdsToSwap` is an array?

Reply:
ahh, I see it was eliminated in a later commit https://github.com/Layr-Labs/eigenlayer-contracts/pull/142/commits/af36906f1f085006984375572a5b711f804a7331